scripts/cnn_variant_wdl/cram2filtered.wdl

```diff
@@ -0,0 +1,473 @@
+                interval_list = calling_interval,
+                gatk_docker = gatk_docker,
+                gatk_override = gatk_override,
+                preemptible_attempts = preemptible_attempts,
+                extra_args = extra_args,
+                disk_space_gb = disk_space_gb
+        }
+
+        call CNNScoreVariants {
+            input:
+                input_vcf = RunHC4.raw_vcf,
+                input_vcf_index = RunHC4.raw_vcf_index,
+                bam_file = RunHC4.bamout,
+                bam_file_index = RunHC4.bamout_index,
+                architecture_json = architecture_json,
+                architecture_hd5 = architecture_hd5,
+                reference_fasta = reference_fasta,
+                tensor_type = tensor_type,
+                inference_batch_size = inference_batch_size,
+                transfer_batch_size = transfer_batch_size,
+                reference_dict = reference_dict,
+                reference_fasta_index = reference_fasta_index,               
+                output_prefix = output_prefix,
+                interval_list = calling_interval,
+                gatk_override = gatk_override,
+                gatk_docker = gatk_docker,
+                preemptible_attempts = preemptible_attempts,
+                mem_gb = mem_gb
+        }
+    }
+
+    call MergeVCFs as MergeVCF_HC4 {
+        input: 
+            input_vcfs = CNNScoreVariants.cnn_annotated_vcf,
+            output_prefix = output_prefix,
+            gatk_override = gatk_override,
+            preemptible_attempts = preemptible_attempts,
+            gatk_docker = gatk_docker
+    }
+ 
+    call MergeBamOuts {
+        input:
+            bam_outs = RunHC4.bamout,
+            output_prefix = output_prefix,
+            ref_fasta = reference_fasta,
+            ref_dict = reference_dict,
+            ref_fai = reference_fasta_index,            
+            gatk_override = gatk_override,
+            preemptible_attempts = preemptible_attempts,
+            gatk_docker = gatk_docker                  
+    }
+
+
+    output {
+        MergeVCF_HC4.*
+        MergeBamOuts.*
+    }
+
+}
+
+
+task CramToBam {
+    File reference_fasta
+    File reference_fasta_index
+    File reference_dict
+    File cram_file
+    String output_prefix
+
+    # Runtime parameters
+    Int? mem_gb
+    Int? preemptible_attempts
+    Int? disk_space_gb
+    Int? cpu 
+    Boolean use_ssd = true
+
+    # You may have to change the following two parameter values depending on the task requirements
+    Int default_ram_mb = 16000
+    # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).
+    Int default_disk_space_gb = 200
+
+    # Mem is in units of GB but our command and memory runtime values are in MB
+    Int machine_mem = if defined(mem_gb) then mem_gb *1000 else default_ram_mb
+    Int command_mem = machine_mem - 1000
+
+command <<<
+  ls -ltr ${cram_file} ${reference_fasta} &&
```

Comment:
This is total shenanigans -- did you get it from Firecloud?

Reply:
Yup.